src/Streamly/Internal/Data/Stream/IsStream/Combinators.hs

```diff
@@ -24,6 +21,8 @@ module Streamly.Internal.Data.Stream.Combinators
     )
 where
 
+#include "inline.hs"
```

Comment:
It is better to add any includes at the top of file (though it doesn't matter for this particular file.)

Reply:
In your refactoring you moved the includes next to imports, and that's how I started doing that. Did you do that by mistake? Anyway, I like it next to the imports because these also serve the same purpose as imports. It should be safe to do it here, we may only need them earlier than this if we are using the definitions in exports.